test/definition/ApiDefinitionFactorySpec.scala

```diff
@@ -49,6 +51,8 @@ class ApiDefinitionFactorySpec extends UnitSpec {
         MockAppConfig.apiStatus(Version2) returns "ALPHA"
         MockAppConfig.endpointsEnabled(Version1) returns true
         MockAppConfig.endpointsEnabled(Version2) returns true
+        MockAppConfig.deprecationFor(Version1).returns(NotDeprecated.valid).anyNumberOfTimes()
+        MockAppConfig.deprecationFor(Version2).returns(NotDeprecated.valid).anyNumberOfTimes()
```

Comment:
Could wrap this in a foreach to make it a bit cleaner. Something like this:

```suggestion
Seq(Version1, Version2).foreach {version =>
          MockAppConfig.apiStatus(version) returns "ALPHA"
          MockAppConfig.endpointsEnabled(version) returns true
          MockAppConfig.deprecationFor(version).returns(NotDeprecated.valid).anyNumberOfTimes()
        }
```